TodoWithControllers/Program.cs

```diff
@@ -1,24 +1,26 @@
-using System.Threading.Tasks;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
 
-namespace Todos
+namespace TodoWithControllers
 {
-    class Program
+    public class Program
```

Comment:
Also no.

Reply:
Then I have to create test project for each API project, it's OK 😕 .